packages/phone-number-privacy/common/src/utils/authentication.ts

```diff
@@ -34,7 +34,14 @@ export async function authenticateUser(
   }
 
   if (authMethod && authMethod === AuthenticationMethod.ENCRYPTION_KEY) {
-    const registeredEncryptionKey = await getDataEncryptionKey(signer, contractKit)
+    let registeredEncryptionKey
+    try {
+      registeredEncryptionKey = await getDataEncryptionKey(signer, contractKit)
+    } catch (error) {
+      logger.error('Failed to retrieve DEK: ' + error.message)
+      logger.warn('Assuming request is authenticated')
```

Comment:
Was this intentional?

Reply:
Yes, I should call it out in the comments, but the assumption here is that an exception will _only_ be thrown if there's a failure to reach the full-node (forno). I don't believe that this will throw an exception if the DEK isn't registered but I should double check.

Reply:
But why should we assume the request to be authenticated?

Reply:
We don't want to block the user due to an outage where possible. I realize this gives attackers an incentive, but we should be able to out-scale them if we're doing our job right :)